eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -149,13 +150,13 @@ jobs:
         displayName: "Install dependencies"
 
       - script: |
-          node common/scripts/install-run-rush.js build --verbose $(GeneratedPackageTargets)
+          node common/scripts/install-run-rush.js build --verbose $(GeneratedPackageTargetsTo) $(GeneratedPackageTargetsFrom)
         displayName: "Build libraries"
       - script: |
-          node common/scripts/install-run-rush.js build:test --verbose $(GeneratedPackageTargets)
+          node common/scripts/install-run-rush.js build:test --verbose $(GeneratedPackageTargetsTo) $(GeneratedPackageTargetsFrom)
```

Comment:
I think `build:test` should just need `$(GeneratedPackageTargetsFrom)`, since I don't think package tests depend on the tests of their dependencies.

Reply:
yup. done